labs/accessing-optimizely-event-data/README.md

```diff
@@ -0,0 +1,140 @@
+# Accessing Optimizely Enriched Event Data
```

Comment:
Maybe the title should callout that this is using a CLI?

"CLI for Accessing Optimizley Enriched Event Data"
"Access Optimizely Enriched Event Data using a CLI"
"CLI access to Optimizely Enriched Event Data"

Reply:
Might also consider updating the slug to include CLI as well.

Reply:
Good point--fixed.